Mathlib/Algebra/Module/DirectLimitAndTensorProduct.lean

```diff
@@ -0,0 +1,96 @@
+/-
+Copyright (c) 2023 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+
+import Mathlib.Algebra.DirectLimit
+
+/-!
+# Tensor product and direct limits commute with each other.
+Given a family of `R`-modules `Gᵢ` with a family of compatible `R`-linear maps `fᵢⱼ : Gᵢ → Gⱼ` for
+every `i ≤ j` and another `R`-module `M`, we have `(limᵢ Gᵢ) ⊗ M` and `lim (Gᵢ ⊗ M)` are isomorphic
```

Comment:
The result is presumably also true with `M` on the right, like it is for direct sums?

Reply:
Yes, I added the other direction at the end of this file.